src/components/footer/index.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { Menu } from "../";
+import "./styles.css";
+
+const additionalMenu = [
+  { text: "Pure UI", url: "https://rauchg.com/2015/pure-ui" },
+  { text: "Documentation", url: "https://reactjs.org/docs/" },
+  {
+    text: "PropTypes",
+    url: "https://reactjs.org/docs/typechecking-with-proptypes.html"
+  },
+  {
+    text: "Create react app",
+    url: "https://github.com/facebook/create-react-app/"
+  },
+  { text: "github", url: "https://github.com/alinepickler/my-project.git" }
+];
+
+const first = additionalMenu[0];
```

Comment:
this is a dead code, we don't need it anymore. pls remove

Reply:
Fixed =)